docs/migration.md

```diff
@@ -47,23 +47,32 @@ public class MyApplication extends FlutterApplication {
     MobileCore.setLogLevel(LoggingMode.DEBUG);
     MobileCore.setWrapperType(WrapperType.FLUTTER);
 
-    try {
-      Identity.registerExtension();
-      Lifecycle.registerExtension();
-      Signal.registerExtension();
-      Assurance.registerExtension();
+-    try {
+-     Identity.registerExtension();
+-     Lifecycle.registerExtension();
+-     Signal.registerExtension();
+-     Assurance.registerExtension();
 -     UserProfile.registerExtension();
 -     Analytics.registerExtension();
 -     Target.registerExtension();
 -     Places.registerExtension();
 -     Campaign.registerExtension();
-      MobileCore.start(new AdobeCallback () {
-          @Override
-          public void call(Object o) {
-            MobileCore.configureWithAppID("yourAppID");
-         }
-      });
-    } catch (InvalidInitException e) {
+-      MobileCore.start(new AdobeCallback () {
+-          @Override
+-         public void call(Object o) {
+-            MobileCore.configureWithAppID("yourAppID");
+-         }
+-      });
+-    } catch (InvalidInitException e) {
+
+  List<Class<? extends Extension>> extensions = Arrays.asList(
+      com.adobe.marketing.mobile.Identity.EXTENSION,
```

Comment:
Can you change it to just Identity.EXTENSION as it does not include EdgeIdentity?